UserManagementSystem/Seeds/SeedersRunner.CS

```diff
@@ -0,0 +1,18 @@
+using UserManagementSystem.Seeds;
+using UserManagementSystem.Src.Infrastructure.Db;
+
+public class SeedersRunner {
+    private readonly ApplicationDbContext _dbContext;
+    private readonly IPasswordHashService _passwordHashService;
+
+    public SeedersRunner(ApplicationDbContext dbContext, IPasswordHashService passwordHashService)
+    {
+        _dbContext = dbContext;
+        _passwordHashService = passwordHashService;
+    }
+
+    public void Run()
+    {
+        // UserSeeder.SeedUsers(_dbContext, _passwordHashService);
```

Comment:
uncomment this line if you need to seed your database, comment it back because it will create new users every time, until we make the email a unique field this will not be a problem!